### PR TITLE
BSP updated.

### PR DESCRIPTION
**Pull request by Xarpagon**

### CODE REVIEW

File: BrokerLock/BrokerLock.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
```
Comment (Lurkerpas) [inline BrokerLock/BrokerLock.h:2]: This file is a part of the TASTE Leon3 Runtime

File: BrokerLock/BrokerLock.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
```
Comment (Lurkerpas) [inline BrokerLock/BrokerLock.h:4]: 2022

File: BrokerLock/BrokerLock.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
```
Comment (Lurkerpas) [inline BrokerLock/BrokerLock.h:9]: GPL. Maybe just better copy the header from Perfmon.c.

File: BrokerLock/BrokerLock.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+#include <BrokerLock.h>
+
+#include <rtems.h>
+
+extern rtems_id brokerLock;
+
+void
+Broker_acquire_lock()
+{
+    rtems_semaphore_obtain(brokerLock, RTEMS_WAIT, RTEMS_NO_TIMEOUT);
```
Comment (Lurkerpas) [inline BrokerLock/BrokerLock.c:32]: Just a question -> you told me that for the UART driver there were spurious wakeups, right? Shouldn't we protect against them here (loop and == RTEMS_SUCCESFFUL?)

Comment (Xarpagon) [reply BrokerLock/BrokerLock.c:32]: ack

File: BrokerLock/BrokerLock.c [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
```
[the remark this answers is not in the document]
Comment (Xarpagon) [reply BrokerLock/BrokerLock.c:6]: ack

File: BrokerLock/BrokerLock.h (hunk 3 above)
Comment (Xarpagon) [reply BrokerLock/BrokerLock.h:9]: ack

File: BrokerLock/BrokerLock.h (hunk 2 above)
Comment (Xarpagon) [reply BrokerLock/BrokerLock.h:4]: ack

**Review by Lurkerpas — DISMISSED** [review]

**Pull request merged**